generators/app/templates/src/StarterKit/Api/Controllers/DbExamplesController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
```

Comment:
Als dit  bestand enkel dient voor het uittesten van de logging bij het uivoeren van DB-queries, dan is dit eerder iets voor een testproject of kan hiervoor beter een unit- of integrationtest geschreven worden. Voor Starterkit zelf levert dit geen meerwaarde op en zal dit waarschijnlijk ook telkens verwijderd worden bij de aanvang van een project. In deze vorm zou ik dit verwijderen.